src/injected/content/inject.js

```diff
@@ -117,12 +122,24 @@ function checkInjectable() {
   return res;
 }
 
-function inject(code) {
+function inject(item) {
   const script = document::createElementNS(NS_HTML, 'script');
+  // Firefox ignores sourceURL comment when a syntax error occurs so we'll print the name manually
+  let onError;
+  if (bridge.isFirefox) {
+    onError = e => {
+      if (e.error.stack::stringIncludes(browser.runtime.getURL('/sandbox'))) {
```

Comment:
Will `e.error.stack` be empty or something other than a string here?

Reply:
Added a check in case the page broken it somehow.